.github/pull_request_template.md

```diff
@@ -15,10 +15,35 @@
 <!-- Describe what manual testing you did (if no testing was done, explain why). -->
 
 ### Automated Testing
-<!--If you added/updated unit/integration/validation tests, describe what cases they cover and do not cover. -->
+<!-- Ensure there are unit/integration/validation tests added (if possible); describe what cases they cover and do not cover. -->
+* Test types added/modified:
+    * Unit
+    * Integration (Go Framework)
+    * Integration (v2prov Framework)
+    * Validation (Go Framework)
+    * Other - Explain: _EXPLAIN_
+    * None
+    * _REMOVE NOT APPLICABLE OPTIONS AND THIS LINE_
```

Comment:
Basically, I'm using _ITALICS ALL CAPS_ to have instructions.
This particular one gives instructions to remove bullet points above that are not applicable.
I thought about doing checkboxes instead - which would be a better fit but currently GH treats it as tasks. Looks like it will _soon_ be addressed though - we can then change to checkboxes.

I agree the wording is a bit confusing, so I will change it to:
```suggestion
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
```